Bowling/NUnitTestBowling/FrameNoBonusRollFactoryTests.cs

```diff
@@ -0,0 +1,46 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using BowlingApp;
+using NUnit.Framework;
+
+namespace NUnitTestBowling
+{
+    class FrameNoBonusRollFactoryTests
+    {
+        private FrameNoBonusRollFactory _subject;
+
+        public FrameNoBonusRollFactoryTests()
+        {
+            _subject = new FrameNoBonusRollFactory();
+        }
+
+        [Test]
+        public void Create_GivenStrikeIn5thFrame_ReturnsScoreOf5thFrameBeforeScoreUpdates()
+        {
+            var result = _subject.Create(5, "4/436/32X63XX9/53", 8);
```

Comment:
With regard to my previous comment about `placeCount` being unnecessary, lets look at this as an example:

This test case is to verify the score of frame number 5.  It's possible to "parse" this string to find the 5th frame:

Frame 1: 4 /
Frame 2: 4 3
Frame 3: 6 /
Frame 4: 3 2
Frame 5: X

Essentially, every frame has 2 rolls unless you roll a strike, or you're on the 10th frame and bowling well.  When the bowling marks are entered into the program, it should be parsed to suit that logic.

Reply:
When I removed the placeCount dependency I ended up writing a function that parsed up everything frame by frame.  This way I'm not passing around the entire bowling string.  Wish I had done that to begin with.   Like in my previous comment this would have solved a bunch of my design issues.